keys/storage/filestorage/main.go

```diff
@@ -124,47 +126,81 @@ func (s FileStore) nameToPaths(name string) (pub, priv string) {
 	return path.Join(s.keyDir, pubName), path.Join(s.keyDir, privName)
 }
 
-func writeInfo(path string, info keys.Info) error {
-	return write(path, info.Name, info.PubKey.Bytes())
-}
-
 func readInfo(path string) (info keys.Info, err error) {
-	var data []byte
-	data, info.Name, err = read(path)
+	f, err := os.Open(path)
```

Comment:
what about closing a file? `defer f.Close()` m?

Reply:
this didn't get addressed

Reply:
It did get addressed

Reply:
was looking at wrong commit, my bad